assets/bingo/lang/de_de.json

```
@@ -1,17 +1,364 @@
+	"bingo.lobby.settings.player.save.new.success": "Neue Voreinstellung “%s” erfolgreich gespeichert",
+	"bingo.lobby.settings.player.save.overwrite.success": "Voreinstellung “%s” erfolgreich mit neuen Einstellungen überschrieben.",
+
+	"bingo.lobby.settings.player.load.sign.line1": "Spieler",
+	"bingo.lobby.settings.player.load.sign.line2": "",
+	"bingo.lobby.settings.player.load.sign.line3": "Auswahl / Ansicht",
+	"bingo.lobby.settings.player.load.sign.line4": "Voreinstellung",
+	"bingo.lobby.settings.player.load.title": "Lade Spieler Voreinstellung",
+	"bingo.lobby.settings.player.load.description": "Zu ladende Voreinstellung:",
+	"bingo.lobby.settings.player.load.new.success": "Voreinstellung “%s” erfolgreich geladen",
+
+	"bingo.lobby.settings.player.configuration.empty": "Leer",
+	"bingo.lobby.settings.player.configuration.night_vision": "Nachtsicht",
+	"bingo.lobby.settings.player.configuration.boats": "Nachtsicht + 'Boats Boats Boats'",
+	"bingo.lobby.settings.player.configuration.elytra": "Nachtsicht + Elytra",
+	"bingo.lobby.settings.player.configuration.frostwalker": "Nachtsicht + Eisläufer",
+	"bingo.lobby.settings.player.configuration.underwater": "Nachtsicht + Unterwasseratmung + Wasserläufer",
+
+	"bingo.lobby.settings.preferences.sign.line1": "Spieler",
+	"bingo.lobby.settings.preferences.sign.line2": "Einstellungen",
+	"bingo.lobby.settings.preferences.sign.line3": "ändern",
+	"bingo.lobby.settings.preferences.sign.line4": "",
+
+	"bingo.lobby.settings.pregen_behavior.sign.line1": "Landschaft",
+	"bingo.lobby.settings.pregen_behavior.sign.line2": "vorzeitig",
+	"bingo.lobby.settings.pregen_behavior.sign.line3": "erzeugen",
+	"bingo.lobby.settings.pregen_behavior.sign.on": "Bei Spielstart",
+	"bingo.lobby.settings.pregen_behavior.sign.off": "Bei Generierung",
+	"bingo.lobby.settings.pregen_behavior.info.on_game_start.prefix": "Bei Spielstart: ",
+	"bingo.lobby.settings.pregen_behavior.info.on_game_start": "Die Landschaft rund um den Spawn wird erzeugt, sobald Du auf das \"Spiel starten\" Schild klickst. Dies führt zu zusätzlicher Wartezeit beim Spielstart.",
+	"bingo.lobby.settings.pregen_behavior.info.on_card_generation.prefix": "Bei Generierung: ",
+	"bingo.lobby.settings.pregen_behavior.info.on_card_generation": "Die Erzeugung der Landschaft beginnt 5 Sekunden nach der Generierung der Bingo-Karte. \nBeachte bitte, dass dies Lag in der Lobby verursachen kann, was in einigen Fällen zu Verlangsamung der Funktionalität in der Lobby führen kann.",
+	"bingo.lobby.settings.pregen_behavior.turn_on": "Wechsel zu “Bei Spielstart”",
+	"bingo.lobby.settings.pregen_behavior.turn_off": "Wechsel zu “Bei Generierung”",
```

```suggestion
	"bingo.lobby.settings.pregen_behavior.info.on_game_start": "Die Landschaft rund um den Spawn wird erzeugt, sobald Du auf das „Spiel starten“ Schild klickst. Dies führt zu zusätzlicher Wartezeit beim Spielstart.",
	"bingo.lobby.settings.pregen_behavior.info.on_card_generation.prefix": "Bei Generierung: ",
	"bingo.lobby.settings.pregen_behavior.info.on_card_generation": "Die Erzeugung der Landschaft beginnt 5 Sekunden nach der Generierung der Bingo-Karte. \nBeachte bitte, dass dies Lag in der Lobby verursachen kann, was in einigen Fällen zu Verlangsamung der Funktionalität in der Lobby führen kann.",
	"bingo.lobby.settings.pregen_behavior.turn_on": "Wechsel zu „Bei Spielstart“",
	"bingo.lobby.settings.pregen_behavior.turn_off": "Wechsel zu „Bei Generierung“",
```
German quotation marks